webapp/providers/github.py

```diff
@@ -61,7 +71,28 @@ def webhook():
                 project=data["repository"]["name"],
             )
         elif flask.request.headers["X-Github-Event"] == "pull_request":
-            print("pull_requests event received")
+            if data["action"] in ["opened", "reopened", "closed"]:
+                pulls_url = data["repository"]["pulls_url"]
+
+                try:
+                    response = api_session.get(pulls_url[:-9])
+                    json = process_response(response)
+
+                    fields = {
+                        "open_prs_count": len(json),
+                        "latest_opened_prs": data["sender"]["login"],
+                        "latest_title_prs": data["pull_request"]["title"],
+                    }
+
+                    payload_to_influx(
+                        measurement="open_prs",
+                        fields=fields,
+                        timestamp=data["pull_request"]["updated_at"],
+                        organisation=data["organization"]["login"],
+                        project=data["repository"]["name"],
+                    )
+                except Exception:
+                    pass
```

Comment:
You could return a response with an error code, just to have some visibility on possible errors.